homeassistant/components/brother/sensor.py

```diff
@@ -66,10 +81,30 @@ def state(self):
     def device_state_attributes(self):
         """Return the state attributes."""
         if self.kind == ATTR_DRUM_REMAINING_LIFE:
-            self._attrs["remaining_pages"] = self.printer.data.get(
+            self._attrs[ATTR_REMAINING_PAGES] = self.printer.data.get(
                 ATTR_DRUM_REMAINING_PAGES
             )
-            self._attrs["counter"] = self.printer.data.get(ATTR_DRUM_COUNTER)
+            self._attrs[ATTR_COUNTER] = self.printer.data.get(ATTR_DRUM_COUNTER)
+        elif self.kind == ATTR_BLACK_DRUM_REMAINING_LIFE:
+            self._attrs[ATTR_REMAINING_PAGES] = self.printer.data.get(
+                ATTR_BLACK_DRUM_REMAINING_PAGES
+            )
+            self._attrs[ATTR_COUNTER] = self.printer.data.get(ATTR_BLACK_DRUM_COUNTER)
```

Comment:
Setting the constant according to `self.kind`:
```suggestion
        elif self.kind == ATTR_BLACK_DRUM_REMAINING_LIFE:
            remaining_pages = ATTR_BLACK_DRUM_REMAINING_PAGES
            drum_counter = ATTR_BLACK_DRUM_COUNTER
```